bokehjs/src/coffee/renderer/glyph/image_url.coffee

```diff
@@ -23,15 +27,20 @@ class ImageURLView extends Glyph.View
 
       if need_load[i]
         img = new Image()
+        img.onerror = do (i) =>
+          return () =>
+            if @retry_attempts[i] > 0
+              setTimeout(@renderer.request_render, @mget('retry_timeout'))
```

Comment:
This seems a bit roundabout. Is it not possible to just trigger the `img` to try a reload? Maybe an anonymous func that re-sets `img.src`?